src/js/components/modals/UpdateConfigModal.js

```diff
@@ -0,0 +1,255 @@
+          <button
+            disabled={pendingRequest}
+            className="button button-success flush-bottom button-wide"
+            onClick={this.handleModalClose}>
+            <i className="icon icon-sprite icon-sprite-mini
+              icon-sprite-mini-white icon-check-mark" />
+          </button>
+        </div>
+      );
+    }
+
+    return (
+      <div className="button-collection flush-bottom">
+        <button className="button button-large flush"
+          onClick={function () {}}>
+          Back
+        </button>
+        <button disabled={hasFormErrors}
+          className="button button-large button-success flush-bottom"
+          onClick={this.handleConfigSave}>
+          Save Configuration
+        </button>
+      </div>
+    );
+  }
+
+  getPackageConfigurationModalContents(service) {
+    // TODO: The currentConfiguration needs to be populated with the package's
+    // current configuration.
+    // JIRA Task DCOS-6638
+    let currentConfiguration = {};
+    let {name, version} = service.get('package');
+    let icon = service.getIcons()['icon-small'];
+    let schema = service.get('config');
+
+    return (
+      <AdvancedConfig
+        model={currentConfiguration}
+        packageIcon={icon}
+        packageName={name}
+        packageVersion={version}
+        schema={schema}
+        onChange={this.handleAdvancedFormChange}
+        getTriggerSubmit={this.getConfigSubmit} />
+    );
+  }
+
+  getUpdateSuccessContent() {
+    let {servicePackage} = this.props;
+
+    let notes = Util.findNestedPropertyInObject(
+      servicePackage.getPackage(),
+      'postInstallNotes'
+    );
+
+    let parsedNotes = StringUtil.parseMarkdown(notes);
+
+    return (
+      <div className="horizontal-center modal-content-inner container
+        container-pod container-pod-short text-align-center">
+        <h2 className="flush-top short-bottom">Success!</h2>
+        <div style={{width: '100%', overflow: 'auto', wordWrap: 'break-word'}}
+          dangerouslySetInnerHTML={parsedNotes} />
+      </div>
+    );
+  }
+
+  render() {
+    let {updateError, updateSuccess} = this.state;
+
+    let modalClasses = classNames('modal', {
+      'modal-large': updateError == null && !updateSuccess,
+      'modal-narrow': !!updateError || updateSuccess
+    });
+
+    return (
+      <Modal
+        backdropClass="modal-backdrop"
+        bodyClass=""
+        footer={this.getPackageConfigurationModalFooter()}
+        innerBodyClass="flush-top flush-bottom"
+        maxHeightPercentage={1}
+        modalClass={modalClasses}
+        modalWrapperClass="multiple-form-modal"
+        onClose={this.handleModalClose}
+        open={this.props.open}
+        showCloseButton={false}
+        showFooter={true}>
+        {this.getModalContents()}
+      </Modal>
+    );
+  }
+}
+
+UpdateConfigModal.propTypes = {
+  service: React.PropTypes.instanceOf(Service).isRequired
```

Comment:
It's both! `service` is the instance of `Framework` and `servicePackage` is an instance of `UniversePackage`, I'm just following the naming convention from the `Install` modal but it would make sense to name it `framework` and `package` :)